egs/pashto/v1/local/chain/run_nn.sh

```diff
@@ -0,0 +1,195 @@
+#!/bin/bash
```

Comment:
Filename? Are the files in other recipes really called run_nn.sh?
By custom, we call them run_<architecture>.sh, for example run_tdnn_lstm.sh or run_cnn.sh and so on.